labchecker/plugins/jobs/image.py

```diff
@@ -0,0 +1,19 @@
+from labcheck import Plugin
+import commands
+
+
+class ImagePlugin(Plugin):
+
+    ptype = 'jobs'
+    name = 'image'
+
+    def check(self):
+        self.failed = False
+        self.reasons = []
+        # find 'cirros' in openstack image list
+        image_check = 'openstack --os-cloud %s image list ' \
+                      '-c "Name" -c "Status" -f value | grep cirros' % self.cloud
```

Comment:
There are public cirros images in some public clouds, so we should use more special name for it, and update zuul jobs to apply it.